README.md

```diff
@@ -147,6 +147,7 @@ bash
 
 - [ ] *New content*
 - [ ] *Testimonial section*
+- [ ] *likes icon to check which continent is most visited*
```

Comment:
- [ ] great job on this milestone please The README contains an "Acknowledgements" section or similar that gives appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/).please give appropriate credit to the author .

Reply:
Dear @ademibowale 
thank you so much for your kind reply and explanation,
I modified the acknowledgements section as you suggested,
have a beautiful week,
Best wishes

Reply:
@IvonneBenitesRodriguez you are welcome madam